### PR TITLE
Re-enable `pod lib lint` for `watchOS`

### PR DESCRIPTION
**Pull request by NachoSoto**
This was fixed by https://github.com/CocoaPods/CocoaPods/issues/11558 in https://github.com/CocoaPods/CocoaPods/releases/tag/1.12.0

### CODE REVIEW

**Review by aboedo — APPROVED** [review]
awesome!! Finally!

**Comment by NachoSoto** [comment]
Hmm it's not using the latest version:
```
CocoaPods 1.12.0 is available.
To update use: `gem install cocoapods`
```
Looking into updating it.

File: .circleci/config.yml [hunk 1]
```diff
@@ -79,6 +79,7 @@ commands:
   run-ios-tests:
     steps:
       - checkout
+      - install-gems
```
Comment (NachoSoto) [inline .circleci/config.yml:82]: This was missing.

**Comment by NachoSoto** [comment]
OMG this is still not working 🤦🏻

**Comment by NachoSoto** [comment]
Closing this until CocoaPods fixes it.

**Pull request closed without merging**